lib/scsi_defs.h

```diff
@@ -80,5 +85,21 @@ namespace scsi_defs {
     uint8_t vendorSpecific : 2;
   } ControlByte;
 
+  //SCSI Reference Manual Table 202 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  typedef struct TestUnitReadyCommand {
```

Comment:
The `typedef struct` pattern isn't needed in C++; you can just declare the struct the same way you would a class.